website/docs/cli/commands/providers/index.mdx

```diff
@@ -15,4 +15,17 @@ This command also has several subcommands with different purposes.
 
 ## Usage
 
-Usage: `tofu providers`
+Usage:
+```
+$ tofu providers
+
+Providers required by configuration:
+.
+└── module.submodule
```

Comment:
I'm not a huge fan of this way of displaying trees because it's not very accessible. Maybe in the docs we could use a different way of conveying this information?

Reply:
This is the raw text output from the command as it exists today.